frontend/eda/rulebook-activations/ActivationInstanceDetails.tsx

```diff
@@ -0,0 +1,117 @@
+import { CodeBlock, CodeBlockCode, PageSection, Skeleton, Stack } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  PageDetail,
+  PageDetails,
+  PageHeader,
+  PageLayout,
+  PageTab,
+  PageTabs,
+  Scrollable,
+} from '../../../framework';
+import { formatDateString } from '../../../framework/utils/formatDateString';
+import { useGet } from '../../common/crud/useGet';
+import { RouteObj } from '../../Routes';
+import { API_PREFIX } from '../constants';
+import { EdaActivationInstance } from '../interfaces/EdaActivationInstance';
+import { EdaActivationInstanceLog } from '../interfaces/EdaActivationInstanceLog';
+import { PageDetailsSection } from '../common/PageDetailsSection';
+import React from 'react';
+import { EdaRulebookActivation } from '../interfaces/EdaRulebookActivation';
+
+export function ActivationInstanceDetails() {
+  const { t } = useTranslation();
+  const params = useParams<{ id: string }>();
+  const { data: activationInstance } = useGet<EdaActivationInstance>(
+    `${API_PREFIX}/activation-instances/${params.id ?? ''}/`
+  );
+
+  const { data: activationInstanceLog } = useGet<EdaActivationInstanceLog>(
+    `${API_PREFIX}/activation-instances/${params.id ?? ''}/logs/`
+  );
+
+  // TODO - remove when the endpoint return the activation name
+  const { data: activation } = useGet<EdaRulebookActivation>(
+    `${API_PREFIX}/activations/${activationInstance?.activation ?? ''}/`
+  );
+
+  const renderActivationDetailsTab = (
+    activationInstance: EdaActivationInstance | undefined
+  ): JSX.Element => {
+    return (
+      <Scrollable>
+        <PageDetails>
+          <PageDetail label={t('Name')}>
+            {activationInstance?.name || `Instance ${activationInstance?.id || ''}`}
+          </PageDetail>
+          <PageDetail label={t('Activation status')}>{activationInstance?.status || ''}</PageDetail>
+          <PageDetail label={t('Start date')}>
+            {activationInstance?.started_at ? formatDateString(activationInstance?.started_at) : ''}
+          </PageDetail>
+          <PageDetail label={t('End date')}>
+            {activationInstance?.ended_at ? formatDateString(activationInstance?.ended_at) : ''}
+          </PageDetail>
+        </PageDetails>
+        <PageDetailsSection>
+          <PageDetail label={t('Output')}>
+            <CodeBlock>
```

Comment:
I think we decided to use monaco editor for variables/code blocks.  We should be consistent, and if we need to discuss this in a more open forum I'm fine with that.